Added an example test for to demonstrate options.setAcceptInsecureCerts(true);

ravigajul opened the pull request:
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->


___

### **PR Type**
Tests


___

### **Description**
- Added a new test method `setAcceptInsecureCerts` in `OptionsTest.java`.
- Configured `ChromeOptions` to accept insecure certificates.
- Included navigation to "https://selenium.dev" to demonstrate the option.



___



### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>OptionsTest.java</strong><dd><code>Add test for accepting insecure certificates in ChromeOptions</code></dd></summary>
<hr>
      
examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java

<li>Added a new test method <code>setAcceptInsecureCerts</code>.<br> <li> Configured <code>ChromeOptions</code> to accept insecure certificates.<br> <li> Included navigation to "https://selenium.dev" to demonstrate the <br>option.<br>


</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1762/files#diff-3eb3ff6008510d0d30b7f1d43c8857bdf29fda42534f08399bccfe1cd323eab5">+13/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

ravigajul committed:
Update OptionsTest.java

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1762) <br/>All committers have signed the CLA.

harsha509 reviewed (approved):
Thank you @ravigajul !

harsha509 commented on examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java:
```diff
@@ -47,5 +47,18 @@ public void setPageLoadStrategyNone() {
+  @Test
+  public void setAcceptInsecureCerts() {
+    ChromeOptions chromeOptions = new ChromeOptions();
+    options.setAcceptInsecureCerts(true);
```
chromeOptions.setAcceptInsecureCerts(true);

ravigajul replied:
My bad. I have updated it to the right reference chromeOptions

harsha509 reviewed (changes requested):
HI @ravigajul ,

Thank you for the PR. 

There is an issue in the code, could you Please review the comments for more details and push the necessary changes

Thanks,
Sri

ravigajul committed:
Update OptionsTest.java

updated to chromeOptions.setAcceptInsecureCerts(true); from options.setAcceptInsecureCerts(true);

ravigajul committed:
Merge branch 'trunk' into patch-1

harsha509 committed:
Merge branch 'trunk' into patch-1

harsha509 commented:
Congratulations on your first contribution to selenium Docs @ravigajul 🎉 🎉 🎉

ravigajul commented:
Thank you for your support and guidance. Feels good. Hoping to contribute more.Sent from my iPhoneOn Jun 23, 2024, at 4:25 AM, Sri Harsha ***@***.***> wrote:﻿
Congratulations on your first contribution to selenium Docs @ravigajul 🎉 🎉 🎉

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
